src/api-explorer/v4-0/ConcurRequest.swagger2.json

```diff
@@ -0,0 +1,4621 @@
+          },
+          "500": {
+            "description": "An unexpected error has prevented the operation",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "503": {
+            "description": "The service is currently unavailable",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/expenses/{expenseUuid}/comments": {
+      "get": {
+        "tags": [
+          "Expense Resource"
+        ],
+        "summary": "Get the list of comments for an existing expected expense",
+        "operationId": "getCommentsByExpenseUsingGET",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "expenseUuid",
+            "in": "path",
+            "description": "The unique identifier of the Expense",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "userId",
+            "in": "query",
+            "description": "The unique identifier of the user",
+            "required": false,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/Comment"
+              }
+            }
+          },
+          "400": {
+            "description": "The request is invalid",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "401": {
+            "description": "Request is not authenticated"
+          },
+          "403": {
+            "description": "Principal is not allowed to perform operation"
+          },
+          "500": {
+            "description": "An unexpected error has prevented the operation",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "503": {
+            "description": "The service is currently unavailable",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/userpolicies": {
+      "get": {
+        "tags": [
+          "Policy Resource"
+        ],
+        "summary": "Get the list of existing Request policies for a given user",
+        "operationId": "getListForUserUsingGET",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "userId",
+            "in": "query",
+            "description": "The unique identifier of the user for whom the list of Request policies will be retrieved",
```

Comment:
The unique identifier of the user for whom the list of Request Policies will be retrieved